samples/built-in-policy/audit-enabling-diagnostic-logs-batch-accounts/README.md

```diff
@@ -0,0 +1,26 @@
+# Audit enabling of diagnostic logs in Batch accounts
```

Comment:
The files in this folder don't follow the naming convention (they have an extra '-' in the name). 
They should be:
- azurepolicy.json
- azurepolicy.parameters.json
- azurepolicy.rules.json


Right now, they are:
- azure-policy.json
- azure-policy.parameters.json
- azure-policy.rules.json